### DISCUSSION TITLE
Polarization tensor/custom tensor

### DISCUSSION THREAD

**Discussion by feyncalccc** [FeynCalc/feyncalc#223] [Q&A] [answered]
For higher spin particles(gravitons for example), the polarization state will be described by a tensor rather than a vector. I'm wondering if there is a standard way to treat such polarization tensors(and higher rank tensors in general), namely to declare a tensor and apply constraints to it (transverse traceless, etc). Thanks a lot!!

**Comment by feyncalccc** [comment]
I think I know the answer. Since polarization vector is a builtin function of feyncalc, one construct a polarization tensor of any rank by putting together polarization vectors. For example, epsilon_\[Mu]\[Nu](p) = epsilon_\[Mu](p) epsilon_\[Nu](p).

Corrections and supplementary answers are still welcome and appreciated.

**Reply by vsht** [reply, marked as answer]
DoPolarizationSums just uses Uncontract, Contract and a bunch of replacement rules to implement the polarization sum formulas. You
can simply write your own function for arbitrary tensors using those commands.
